### PR TITLE
refactor(): more semantic topic names

### PR DESCRIPTION
**Pull request by rokusottervanger**

### CODE REVIEW

File: src/hotword_detector_node.cpp [hunk 1]
```diff
@@ -27,8 +27,8 @@ class HotwordDetectorNode
 
   bool initialize()
   {
-    audio_sub_ = nh_.subscribe("audio", 1000, &HotwordDetectorNode::audioCallback, this);
-    trigger_pub_ = nh_.advertise<std_msgs::String>("trigger", 10);
+    audio_sub_ = nh_.subscribe("microphone", 1000, &HotwordDetectorNode::audioCallback, this);
+    trigger_pub_ = nh_.advertise<std_msgs::String>("hotword_detection", 10);
```
Comment (reinzor) [inline src/hotword_detector_node.cpp]: param is still called trigger_string

**Review by reinzor — DISMISSED** [review]

**Review by LoyVanBeek — APPROVED** [review]

**Comment by rokusottervanger** [comment]
@reinzor, is this OK now?

**Comment by reinzor** [comment]
I think so :)

-Rein

On Tue, May 30, 2017 at 7:32 PM, Rokus Ottervanger <notifications@github.com

**Comment by LoyVanBeek** [comment]
@reinzor can you approve the changes then?

**Comment by MatthijsBurgh** [comment]
@rokusottervanger , Ready to be merged

**Pull request merged**